ehrql/__init__.py

```diff
@@ -32,6 +33,7 @@
     "create_dataset",
     "create_measures",
     "days",
+    "get_dataset_parameter",
```

Comment:
What do you think about just `get_parameter` as a name? I'm not sure "dataset" adds much and its potentially confusing and you can use these with measures (or whatever other things we come up with in future).

Reply:
Sounds good. I originally didn't call it just `get_parameter` because we had other just Parameter things, but those aren't exposed to users. Hadn't occurred to me that they're usuable for measures too.